minio/xml_marshal.py

```diff
@@ -29,9 +29,48 @@
 import io
 
 from xml.etree import ElementTree as s3_xml
+from collections import defaultdict
 
 _S3_NAMESPACE = 'http://s3.amazonaws.com/doc/2006-03-01/'
 
+def xml_to_dict(in_xml):
```

Comment:
This function does not seem to be used

Reply:
You're right. I was supposed to use it to return a dictionary instead of xml from `get_bucket_encryption` api.
I'll fix it.